feeds/nuget/nuget.go

```diff
@@ -0,0 +1,195 @@
+package nuget
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/ossf/package-feeds/feeds"
+)
+
+const (
+	FeedName           = "nuget"
+	feedURL            = "https://api.nuget.org/v3/index.json"
+	catalogServiceType = "Catalog/3.0.0"
+)
+
+type nugetServiceIndex struct {
+	Services []*nugetService `json:"resources"`
+}
+
+type nugetService struct {
+	URI  string `json:"@id"`
+	Type string `json:"@type"`
+}
+
+type catalog struct {
+	Pages []*catalogPage `json:"items"`
+}
+
+type catalogPage struct {
+	URI      string         `json:"@id"`
+	Created  time.Time      `json:"commitTimeStamp"`
+	Packages []*catalogLeaf `json:"items"`
+}
+
+type catalogLeaf struct {
+	URI            string    `json:"@id"`
+	CatalogCreated time.Time `json:"commitTimeStamp"`
+	Type           string    `json:"@type"`
+}
+
+type nugetPackageDetails struct {
+	PackageID string    `json:"id"`
+	Version   string    `json:"version"`
+	Created   time.Time `json:"published"`
+}
+
+var httpClient http.Client = http.Client{
+	Timeout: 10 * time.Second,
+}
+
+func fetchCatalogService() (*nugetService, error) {
+	resp, err := httpClient.Get(feedURL)
+
```

Comment:
As the smallest nit, you could consider removing the spacing between where an error is initialized and where we check it.

Spacing really isn't a big deal to me and is a total nonblocker, but it'll help match the style of the other feeds a bit better.